conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -46,25 +44,34 @@ install:
 
     # Cywing's git breaks conda-build. (See https://github.com/conda-forge/conda-smithy-feedstock/pull/2.)
     - cmd: rmdir C:\cygwin /s /q
-    - appveyor DownloadFile "https://raw.githubusercontent.com/conda-forge/conda-smithy/e976c7e84bb3c4846e7562afd1a42c4b535b51f5/bootstrap-obvious-ci-and-miniconda.py"
-    - cmd: python bootstrap-obvious-ci-and-miniconda.py %CONDA_INSTALL_LOCN% %TARGET_ARCH% %CONDA_PY:~0,1% --without-obvci
+
+    # Use AppVeyor's preinstalled Miniconda.
+    - cmd: if "%CONDA_PY%" == "27" if  "%TARGET_ARCH%" == "x86" set "CONDA_INSTALL_LOCN=C:\\Miniconda"
+    - cmd: if "%CONDA_PY%" == "27" if  "%TARGET_ARCH%" == "x64" set "CONDA_INSTALL_LOCN=C:\\Miniconda-x64"
+    - cmd: if "%CONDA_PY%" == "34" if  "%TARGET_ARCH%" == "x86" set "CONDA_INSTALL_LOCN=C:\\Miniconda3"
+    - cmd: if "%CONDA_PY%" == "34" if  "%TARGET_ARCH%" == "x64" set "CONDA_INSTALL_LOCN=C:\\Miniconda3-x64"
+    - cmd: if "%CONDA_PY%" == "35" if  "%TARGET_ARCH%" == "x86" set "CONDA_INSTALL_LOCN=C:\\Miniconda35"
+    - cmd: if "%CONDA_PY%" == "35" if  "%TARGET_ARCH%" == "x64" set "CONDA_INSTALL_LOCN=C:\\Miniconda35-x64"
+
+    # Add our channels.
+    - cmd: set "OLDPATH=%PATH%"
+    - cmd: set "PATH=%CONDA_INSTALL_LOCN%\\Scripts;%CONDA_INSTALL_LOCN%\\Library\\bin;%PATH%"
```

Comment:
Actually these lines *technically* belong below with the hack. However, pulling things from `conda-forge` seems to fix the bug we were experiencing. So it had to go up here.